GrEmit/GroboIL.cs

```diff
@@ -1996,15 +2000,15 @@ public void Calli(CallingConventions callingConvention, Type returnType, Type[]
                 il.EmitCalli(OpCodes.Calli, callingConvention, returnType, parameterTypes, optionalParameterTypes);
         }
 
-#if !NETSTANDARD2_0
-    /// <summary>
-    ///     Calls the method indicated on the evaluation stack (as a pointer to an entry point) with arguments described by a calling convention.
-    /// </summary>
-    /// <param name="callingConvention">The unmanaged calling convention to be used.</param>
-    /// <param name="returnType">
-    ///     The <see cref="Type">Type</see> of the result.
-    /// </param>
-    /// <param name="parameterTypes">The types of the required arguments to the instruction.</param>
+#if !NETSTANDARD2_0 // see https://apisof.net/catalog/System.Reflection.Emit.ILGenerator.EmitCalli(OpCode,CallingConvention,Type,Type())
+        /// <summary>
+        ///     Calls the method indicated on the evaluation stack (as a pointer to an entry point) with arguments described by a calling convention.
+        /// </summary>
+        /// <param name="callingConvention">The unmanaged calling convention to be used.</param>
+        /// <param name="returnType">
+        ///     The <see cref="Type">Type</see> of the result.
+        /// </param>
+        /// <param name="parameterTypes">The types of the required arguments to the instruction.</param>
         public void Calli(CallingConvention callingConvention, Type returnType, Type[] parameterTypes)
```

Comment:
I think, we need to add a test for this `GroboIL.Calli()` method with native calling convention.

Reply:
Done